boilerplate/babel.config.js

```diff
@@ -1,4 +1,19 @@
 const plugins = [
+  [
+    "babel-plugin-root-import",
```

Comment:
Just so I understand, what makes it so we have to configure babel like this in addition to tsconfig? From discussion in slack it had sounded like tsconfig alone might be enough but I never looked closely

Reply:
Babel has to be configured to tell metro about how this magical path works.

The tsconfig paths alone will make things look like the path works as you just edit syntax, but when running metro won't know about that new path mapping.